ops/buildkite/deploy-content

```diff
@@ -119,7 +119,7 @@ deploy_to_cloudflare() {
         # for branch-specific non-production deploys, use `--branch master`
         cd dist
         # trim \r, they remove previous logs in buildkite
-        env $(grep "^CLOUDFLARE" ../owid-grapher/.env | xargs -0) npx wrangler pages deploy . --project-name "$PROJECT_NAME" 2>&1 | tr -d '\r'
+        env $(grep "^CLOUDFLARE" ../owid-grapher/.env | xargs -0) yarn wrangler pages deploy . --project-name "$PROJECT_NAME" 2>&1 | tr -d '\r'
```

Comment:
@Marigold did you manage to make this work? It should work in https://github.com/owid/ops/pull/122 since it is running from `owid-grapher` but it is here running from `dist` which doesn't have a `package.json`

Reply:
Sorry, I was in a hurry yesterday, messed up testing, requested your review and then made a draft out of it 🤦

I've changed it to `node_modules/wrangler/bin/wrangler.js` and left a comment about `node_modules/.bin/wrangler`. It should work now.

Reply:
no worries! I was commenting in case it's useful (also for me 🙂). I haven't run it in this context but the same command in [deploy-content-preview](https://github.com/owid/owid-grapher/pull/3055/files#diff-78f3f8f68d8a85de432f232a0f3e614525a9e672efad14a36752bc185123e607R167) works as expected, so ✅.